analytics/analytics.properties

```diff
@@ -13,3 +13,9 @@ instance=geOrchestra
 # Timezone to convert datetime sent by UI to UTC
 # (see Canonical ID on http://joda-time.sourceforge.net/timezones.html for possible values)
 localTimezone=Europe/Paris
+
+# List of user to exclude from distinct users statistics (/analytics/ws/distinctUsers)
+# You can add other users to exclude with :
+#protectedUser.uid2=monitoringUser
+protectedUser.uid1=geoserver_privileged_user
```

Comment:
well, it's not a protected user, but an excluded user, no ?
so: `excludedUser.uid1`